src/test/kotlin/three/consulting/epoc/service/TaskServiceIntegrationTest.kt

```diff
@@ -142,4 +142,16 @@ class TaskServiceIntegrationTest : IntegrationTest() {
             .isInstanceOf(UnableToDeleteTaskException::class.java)
             .hasMessage("Cannot delete task, no task found for given id: 1000")
     }
+
+    @Test
+    fun `searching task with project id 1 returns array of task objects`() {
+        val tasks = taskService.findTaskForProjectId(1L)
+        assertThat(tasks).hasSize(2)
```

Comment:
You can/should also check the contents, not just the size. You can do something like 
```kotlin
asserThat(tasks)
    .hasSize(2)
    .anySatisfy {
        assertThat(it.name).isEqualTo("foo")
        asserThat(it.somethingElse).isEqualTo("bar")
    }
```
to check the list members are as they should